src/components/button/index.tsx

```diff
@@ -50,10 +50,10 @@ export function ButtonNextLink({
   className = '',
   children,
   ...props
-}: ButtonProps & LinkType) {
+}: ButtonProps & LinkType & Pick<AType, 'aria-label'>) {
   return (
     <Link {...props}>
-      <a className={and(style.buttonLink, className)}>{children}</a>
+      <a className={and(style.buttonLink, className)} aria-label={props["aria-label"]}>{children}</a>
```

Comment:
This this might rather be rewritten to use `<Link>` without `<a>` child. Current version of `next/link` doesn't require `<a>`-child as the previous versions did. Would simplify types given that `Link` supports aria attrs

Reply:
That would have been nice, [next/link](https://nextjs.org/docs/api-reference/next/link) don't have className so ended up keeping this as is